prow/plugins/milestone/milestone.go

```diff
@@ -125,12 +126,17 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, m
 	milestoneMap := buildMilestoneMap(milestones)
 	milestoneNumber, ok := milestoneMap[proposedMilestone]
 	if !ok {
-		var buffer bytes.Buffer
-		buffer.WriteString("\t 1." + clearKeyword + "\n")
+		slice := make([]string, 0, len(milestoneMap))
 		for k := range milestoneMap {
-			buffer.WriteString("\t 1." + k + "\n")
+			slice = append(slice, k)
+		}
+		sort.Strings(slice)
```

Comment:
we should probably comparison func that is aware of semantic versions

Reply:
OTOH that would not be great for repos using some other milestone format. /shrug

Reply:
> OTOH that would not be great for repos using some other milestone format.

Yeah, I was trying to avoid making this specific to our milestone naming conventions. I don't think the sort ordering is a big deal. This comment only appears if an invalid milestone is supplied and is just meant to show all the available options so I think this sorting is probably ok?

Reply:
sorting versions: still one of the hardest problems in computer science. :)